sdk/rapid/rapid.py

```diff
@@ -413,3 +413,130 @@ def update_subject_permissions(self, subject_id: str, permissions: list[str]):
         raise InvalidPermissionsException(
             "One or more of the provided permissions is invalid or duplicated"
         )
+
+    def create_user(self, user_name: str, user_email: str, user_permissions: list[str]):
```

Comment:
Looks good! The function doesn't seem to handle responses from a user issuing invalid usernames/emails with a clear error message. There are some validation functions in the API code you can steal for this

Reply:
There is an issue with API status code response. For invalid permissions or username/email it returns same code.
I can cover it with if/else statement to return meaningful error anyway. 
I will also add to handle responses when user doesn't have user_admin permissions.

Reply:
Fixed.